libraries/ghc-boot-th/ghc-boot-th.cabal.in

```diff
@@ -56,7 +56,7 @@ Library
         cpp-options: -DBOOTSTRAP_TH
         build-depends:
             ghc-prim
-        hs-source-dirs: @SourceRoot@ ../ghc-internal/src
+        hs-source-dirs: ../ghc-boot-th ../ghc-internal/src
```

Comment:
This is really stupid. For ghc-boot-th this works, because well ../ghc-boot-th is the same. For the synthezied -next package, this works, because we look at the ../ghc-boot-th for the modules.

This TH story is such a PITA.